base/array.jl

```diff
@@ -752,9 +837,45 @@ function findnext(A, v, start::Integer)
     end
     return 0
 end
+"""
+    findfirst(A, v)
+
+Return the linear index of the first element equal to `v` in `A`.
+Returns 0 if `v` is not found.
```

Comment:
maybe also format the 0 as code? since you otherwise escape the zeros consistently.